templates/data_management/object_detail.html

```diff
@@ -80,7 +80,7 @@ <h3>{{ object.name }}</h3>
     {% endif %}
     {% if object.external_object %}
     <dt class="col-sm-3">Download</dt>
-    <dd class="col-sm-9"><a href="{{ request.META.HTTP_NAME }}/external_object/{{ object.external_object.doi_or_unique_name }}:{{ object.external_object.title }}@{{ object.external_object.version }}">link</a></dd>
+    <dd class="col-sm-9"><a href="{{ request.META.HTTP_NAME }}/external_object/{{ object.external_object.other_unique_name }}:{{ object.external_object.title }}@{{ object.external_object.version }}">link</a></dd>
```

Comment:
Should presumably include `identifier`?

Reply:
It will be more complex than that - this is generating the shortcut URL for downloading from Echo. We'll have to use `identifier` if it's defined or use `other_unique_name` if that is defined.

And when `identifier` is used we'll have one URL embedded in another URL...

Reply:
As I just commented above, if it has a doi / other identifier, can we use that directly, and then this would this work as it stands exclusively in the case where `other_unique_name` is non-blank? i.e. provide this download link for an `other_unique_name`, or provide the `identifier` link directly otherwise?

Reply:
Now resolved